source/slang/slang-ir.cpp

```diff
@@ -634,6 +634,13 @@ namespace Slang
         return entryBlock->getParams();
     }
 
+    IRInst* IRGlobalValueWithParams::getFirstNonParamInst()
+    {
+        auto lastParam = getLastParam();
+        if (lastParam == nullptr)
+            return getFirstBlock()->getFirstInst();
+        return lastParam->next;
+    }
```

Comment:
As discussed elsewhere, I *think* this function does what `getFirstOrdinaryInst` already does.